ci/envoy_build_sha.sh

```diff
@@ -1,3 +1,3 @@
 # If you edit the SHA here you must also edit the SHA in .circleci/config.yml.
 
-ENVOY_BUILD_SHA=8839608c2f4b0d7c8311361ee5c8785e543d517a
+ENVOY_BUILD_SHA=44d539cb572d04c81b62425373440c54934cf267
```

Comment:
See comment above ^. You need to edit more places. Also, I think @jmillikin-stripe might still be working out some issues with the image upgrade.